src/PrestaShopBundle/Resources/views/Admin/Module/Includes/see_more.html.twig

```diff
@@ -0,0 +1,34 @@
+{#**
+  * 2007-2018 PrestaShop
+  *
+  * NOTICE OF LICENSE
+  *
+  * This source file is subject to the Open Software License (OSL 3.0)
+  * that is bundled with this package in the file LICENSE.txt.
+  * It is also available through the world-wide-web at this URL:
+  * https://opensource.org/licenses/OSL-3.0
+  * If you did not receive a copy of the license and are unable to
+  * obtain it through the world-wide-web, please send an email
+  * to license@prestashop.com so we can send you a copy immediately.
+  *
+  * DISCLAIMER
+  *
+  * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+  * versions in the future. If you wish to customize PrestaShop for your
+  * needs please refer to http://www.prestashop.com for more information.
+  *
+  * @author    PrestaShop SA <contact@prestashop.com>
+  * @copyright 2007-2018 PrestaShop SA
+  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+  * International Registered Trademark & Property of PrestaShop SA
+  *#}
+<div class="col-12">
+  <p class="text-center">
+    <button type="button" class="btn btn-link see-more" data-category="{{ id }}">
+      {{ 'See more' | trans({}, 'Admin.Modules.Feature') }}
```

Comment:
It already exists in _Admin.Orderscustomers.Feature_ but I guess it should be smarter to make it fall into the _Admin.Actions_ file. Indeed, it is a generic action that can be found elsewhere in the back office.

Reply:
SIR YES SIR!